src/app/containers/Video/videoMetadata.js

```diff
@@ -0,0 +1,32 @@
+import deepGet from '../../helpers/json/deepGet';
+
+const videoObject = 'VideoObject';
+
+const videoMetadata = blocks => {
+  const aresMediaBlocks = deepGet(['model', 'blocks'], blocks);
+  const listContent = [];
+  const metadata = {
+    video: {
+      '@list': listContent,
+    },
+  };
+
+  if (!aresMediaBlocks || aresMediaBlocks.length < 1) {
+    return undefined;
+  }
+  aresMediaBlocks.forEach(block => {
```

Comment:
You can use a filter function here instead, so you don't need to nest a `forEach` with an `if`.